### PR TITLE
add configs that include rspec and graphql for v1.23.0

### PR DESCRIPTION
**Pull request by rebeccafae**
Add configs that include rspec, and rspec+graphql (assuming that no one will need one with only graphql right now)

To test it, follow the [ReadMe instructions](https://github.com/q-centrix/style-guides/blob/main/ruby/README.md#with-every-update) and use the urls from this PR. (Change the endings to `style-guides/rubocop-rspec-1-23-0/ruby/.rubocop-1-23-0-rspec-graphql.yml` or `style-guides/rubocop-rspec-1-23-0/ruby/.rubocop-1-23-0-rspec.yml`)

### CODE REVIEW

File: ruby/.rubocop-1-23-0-rspec.yml [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+Style/VariableInterpolation:
+  Enabled: false
+Style/WhenThen:
+  Enabled: false
+Style/WhileUntilModifier:
+  Enabled: false
+
+
+# for rubocop-rspec
+# Documentation can be found at https://docs.rubocop.org/rubocop-rspec/2.10/cops_rspec.html
```
Comment (albertski) [inline ruby/.rubocop-1-23-0-rspec.yml:331]: Everything looks great. From testing a couple of rspec specs, I saw a lot of failures for:
```
RSpec/MultipleMemoizedHelpers: Example group has too many memoized helpers
```

Some of our specs just need a lot of setup and will cause this error.  Do you think we can ignore this one for now?

I did see a lot of other failures but feel like going forward we should follow the best practices.

Comment (rebeccafae) [reply ruby/.rubocop-1-23-0-rspec.yml:331]: I'm not sure exactly what you're asking/suggesting, I'll Slack you and then summarize over here.

Comment (rebeccafae) [reply ruby/.rubocop-1-23-0-rspec.yml:331]: We all agree to disable this

**Review by albertski — APPROVED** [review]
Looks great.

**Review by jessigu — APPROVED** [review]
used onc-imp [branch](https://github.com/q-centrix/oncology-implementations/pull/800) to test. looks good to me 👍

**Pull request merged**